src/cli/commands/team.js

```diff
@@ -71,23 +81,31 @@ function wrapRequired(callback: CLIFunctionWithParts, requireTeam: boolean, isDe
   };
 }
 
-function wrapRequiredTeam(callback: CLIFunctionWithParts, requireTeam: boolean = true): CLIFunction {
-  return wrapRequired(function(
-    parts: TeamParts,
-    config: Config,
-    reporter: Reporter,
-    flags: Object,
-    args: Array<string>,
-  ): CLIFunctionReturn {
-    if (args.length === 1) {
-      return callback(parts, config, reporter, flags, args);
-    } else {
-      return false;
-    }
-  }, requireTeam);
+function wrapRequiredTeam(
+  callback: CLIFunctionWithParts,
+  requireTeam: boolean = true,
+  subCommandDeprecated?: DeprecationWarning,
+): CLIFunction {
+  return wrapRequired(
+    function(
+      parts: TeamParts,
+      config: Config,
+      reporter: Reporter,
+      flags: Object,
+      args: Array<string>,
+    ): CLIFunctionReturn {
+      if (args.length === 1) {
```

Comment:
`yarn team list` only takes one arg which is either: `<scope>` or `<scope:team>` more than one arg and the command should fail.